contrib/fuzz/oss_fuzz_build.sh

```diff
@@ -37,7 +37,10 @@ compile_fuzzers() {
     done
 }
 
-# This is from https://github.com/AdamKorcz/instrumentation
+# Pull additional fuzzers from the instrumentation repository.
+if [ ! -d "$SRC/instrumentation" ]; then
+    git clone --depth=1 git@github.com/AdamKorcz/instrumentation.git $SRC/instrumentation
```

Comment:
I think there is a typo here; repo should have a `:` after github.com:
```
git@github.com:AdamKorcz/instrumentation.git
```

Reply:
🤦🏼 i do that all the time. lol. Nice catch.